### ISSUE TITLE
Calcurse doesn't redraw while adding a task

### ISSUE THREAD

**Issue by NWuensche** [lfos/calcurse#145]
Hello,

I'm using Calcurse v 4.3.0 on Arch Linux and I have a problem when I change the size of the terminal while I'm adding an entry. Normally, Calcurse redraws when I change the terminal size with i3wm or tmux. However, when I'm adding a task (pressing `a`), Calcurse doesn't redraw properly. I use terminator 1.91 as a terminal emulator and i3 in version 4.15 and tmux 2.7.

Normal behavior when not adding a task when resizing fullscreen terminal to half-size:
![12 09 2018 17 14 11](https://user-images.githubusercontent.com/15856197/45434802-8d3e4300-b6af-11e8-8e5f-debf80437514.png)

Behavior when currently adding a task:
![12 09 2018 17 13 56](https://user-images.githubusercontent.com/15856197/45434749-70097480-b6af-11e8-90dc-f86b5ef6a13b.png)

**Comment by lfos** [comment]
Yes, it's a known bug. It's useful to have it recorded on the tracker, though; thanks!
